css/style.css

```diff
@@ -0,0 +1,461 @@
+body {
+  background-color: #f4f5f6;
+  color: #333;
+  font-family: Arial, Helvetica, sans-serif;
+  overflow-x: hidden;
+}
+
+.text-white {
+  color: white;
+}
+
+.bg-white {
+  background-color: white;
+}
+
+.top-header {
+  height: 95px;
+  background-color: red;
+  padding: 0 10px 15px;
+}
+
+.family-roboto {
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+.container-fluid {
+  max-width: 1310px;
+  margin: 0 auto;
+  padding: 0;
+}
+
+.header-list li {
+  font-size: 18px;
+}
+
+.date {
+  margin-top: 35px;
+}
+
+.nav-sub {
+  margin-top: 30px;
+}
+
+.btn-in:hover {
+  color: white;
+  text-decoration: none;
+}
+
+.btn-sub {
+  padding: 7px 12px;
+  color: red;
+}
+.btn-sub:hover {
+  color: red;
+  text-decoration: none;
+}
+
+.header-nav {
+  padding: 15px 160px 15px 0;
+  border-bottom: 2px solid #d9d9d9;
+  height: 56px;
+  background-color: white;
+}
+.header-nav::before {
+  position: absolute;
+  top: 0;
+  right: 0;
+  content: "";
+  width: 82px;
+  background-color: #f72210;
+  border-left: 12px solid #fff;
+  border-top: 13px solid #f72210;
+}
+
+.nav-list {
+  font-size: 21px;
+  line-height: 10px;
+  font-weight: 700;
+  width: 100%;
+}
+.nav-list li a {
+  color: #000;
+  text-decoration: none;
+}
+.nav-list li a:hover {
+  color: red;
+}
+
+.br-nav-item {
+  border-right: 1px solid #d9d9d9;
+}
+
+.nav-search {
+  top: 3px;
+  right: 105px;
+  width: 60px;
+  height: 49px;
+  font-size: 18px;
+  position: absolute;
+}
+
+.search-icon {
+  top: 20px;
+  clip: auto;
+  width: 20px;
+  right: 80px;
+  height: 20px;
+  font-size: 20px;
+  position: absolute;
+}
+
+.main {
+  padding: 20px 30px;
+}
+
+.block-tittle {
+  color: rgba(3, 3, 3, 0.4);
+  font-size: 16px;
+  padding-bottom: 11px;
+}
+
+.category {
+  padding: 8px 15px 9px;
+  background-color: rgba(255, 255, 255, 0.9);
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  z-index: 1;
+}
+.category a {
+  color: #f72210;
+  font-size: 14px;
+}
+
+.header-article {
+  position: relative;
```

Comment:
`SUGGESTION`

Consider using a bootstrap class instead. _Note that the project is about using bootstrap._